python_modules/dagster/dagster/_core/definitions/partition.py

```diff
@@ -710,6 +714,23 @@ def has_partition_key(
             partitions_def_name=self._validated_name(), partition_key=partition_key
         )
 
+    def get_pending_partition(self, partition_key: str) -> Partition:
+        return Partition(partition_key, partition_key)
+
+    def request_for_adding_partitions(
```

Comment:
What do you think about `build_add_request`? Briefer and helps avoid the possible misunderstanding that "request" is a verb in this context.